docs/developer_guide/running_the_federation.notebook.rst

```diff
@@ -4,7 +4,7 @@
 .. _running_notebook:
 
 **********************************
-Aggregator-Based Workflow Tutorial
+Aggregator-Based Workflow Tutorial (Deprecated)

```

Comment:
This isn't related to Python API, so please revert this change

Reply:
@teoparvanov, I think we should rewrite/delete this as the title is Aggregator based but the the tutorial is using python native API.
https://openfl.readthedocs.io/en/latest/developer_guide/running_the_federation.notebook.html